src/frequenz/sdk/timeseries/_formula_engine/_formula_engine.py

```diff
@@ -45,12 +47,14 @@
 _logger = logging.Logger(__name__)
 
 _operator_precedence = {
-    "(": 0,
-    "/": 1,
-    "*": 2,
-    "-": 3,
-    "+": 4,
-    ")": 5,
+    "max": 0,
+    "min": 1,
```

Comment:
why is `min` and `max` having two different operator precedences?

And it's a global non-constant map, why is it all lower-case and not upper-case? (just asking, trying to understand) :)

Reply:
actually I think it doesn't make much of a difference. But you are right technically they can have the same precedence.